src/pane-element.coffee

```diff
@@ -37,8 +38,21 @@ class PaneElement extends HTMLElement
     handleBlur = (event) =>
       @model.blur() unless @contains(event.relatedTarget)
 
+    handleDragOver = (event) =>
+      event.preventDefault()
+      event.stopPropagation()
+
+    handleDrop = (event) =>
+      event.preventDefault()
+      event.stopPropagation()
+      @getModel().activate()
+      pathsToOpen = _.pluck(event.dataTransfer.files, 'path')
```

Comment:
You could write i think, so you could remove the dependency to `_` (i just saw they are trying that in new packages currently)...
```coffeescript
pathsToOpen = event.dataTransfer.files.map (file) -> file.path
```
See [array.map](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/map)